install_config/certificate_customization.adoc

```diff
@@ -16,8 +16,7 @@ Administrators can configure custom serving certificates for the public host
 names of the {product-title} API and
 xref:../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
 This can be done during an
-xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced
-installation] or configured after installation.
+xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced installation] or configured after installation.
```

Comment:
I like this, however don't feel it call's out "loudly" enough that we should be using Ansible to configure these directives, when possible. I wonder if duplicating content (as a new section) on this document, from the link provided helps to make this point? 

- Note: I am not a fan of duplicating content.